### PR TITLE
Update the version in the Dockerfiles as well

### PR DESCRIPTION
**Pull request by TheZoker**

### CODE REVIEW

File: jira/Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends fontconfig python3 python3-jinja2 tini git \
     && apt-get clean autoclean && apt-get autoremove -y && rm -rf /var/lib/apt/lists/*
 
-ARG VERSION=8.20.11
+ARG VERSION=9.4.0
```
Comment (Hialus) [inline jira/Dockerfile]: Jira Prelive is actually on 9.4.3 now.
Any reason why you don't use that here?

**Review by Hialus — CHANGES_REQUESTED** [review]

File: jira/Dockerfile (hunk 1 above)
Comment (TheZoker) [reply jira/Dockerfile]: I used the same version that Matthias used in his commit today:
https://github.com/ls1intum/Artemis-Local-Setup-Docker/commit/af953b71eb977f8e972689412c4b60d5d3ed588c

This PR is just to bring the dockerfiles in sync with the workflow file again

Comment (Hialus) [reply jira/Dockerfile]: Alright then. Guess this works then. And I'll redirect this question to @Mtze.

**Review by Hialus — DISMISSED** [review]

File: jira/Dockerfile (hunk 1 above)
Comment (Hialus) [reply jira/Dockerfile]: I talked to Matthias and this was indeed a mistake. You can update both instances to 9.4.3.

Comment (Mtze) [reply jira/Dockerfile]: Thank you for pointing that out!

Comment (Mtze) [reply jira/Dockerfile]: Please also update the workflow file to 9.4.3

Comment (TheZoker) [reply jira/Dockerfile]: Done 👍

**Review by Mtze — APPROVED** [review]

**Pull request merged**